src/cascadia/TerminalSettingsModel/GlobalAppSettings.h

```diff
@@ -60,6 +60,11 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         void UnparsedDefaultProfile(const hstring& value);
         void ClearUnparsedDefaultProfile();
 
+        void SetInvertedDisableAnimationsValue(bool invertedDisableAnimationsValue)

```

Comment:
```suggestion
        // TODO GH#9207: Remove this once we have a GlobalAppSettingsViewModel in TerminalSettingsEditor
        void SetInvertedDisableAnimationsValue(bool invertedDisableAnimationsValue)
```
Ugh. I get why we need to do this, but I really wish we didn't have to touch the settings model at all. I guess that's just blocked by https://github.com/microsoft/terminal/issues/9207. I'll live haha.

Reply:
Yeah, I'm totally with you on this. Having to inject those kind of functions since x:Bind does not allow us to do this differently is really annoying.